test/quantization/dbr/test_quantize_dbr.py

```diff
@@ -118,40 +118,81 @@ class TestQuantizeDBRIndividualOps(QuantizeDBRTestCase):
     Tests that DBR quantization covers individual ops
     """
     def test_conv(self):
+        convs = {1: nn.Conv1d, 2: nn.Conv2d, 3: nn.Conv3d}
+
         class M(torch.nn.Module):
-            def __init__(self):
+            def __init__(self, dim):
                 super().__init__()
-                self.conv = torch.nn.Conv2d(1, 1, 1)
+                self.conv = convs[dim](3, 3, 3)
 
             def forward(self, x):
-                x1 = self.conv(x)
-                return x1
+                return self.conv(x)
 
-        m = M().eval()
-        qconfig = torch.quantization.default_qconfig
-        self._test_auto_tracing(m, qconfig, (torch.randn(1, 1, 2, 2),))
+        for dim in range(1, 4):
+            m = M(dim).eval()
+            qconfig = torch.quantization.default_qconfig
+            data = self.img_data_dict[dim][0][0]
+            self._test_auto_tracing(m, qconfig, (data,))
 
     def test_conv_functional(self):
+        convs = {1: F.conv1d, 2: F.conv2d, 3: F.conv3d}
 
         class M(torch.nn.Module):
-            def __init__(self, weight2d, bias2d):
+            def __init__(self, dim, weight):
                 super().__init__()
-                self.weight2d = torch.nn.Parameter(weight2d)
-                self.bias2d = torch.nn.Parameter(bias2d)
-                self.stride2d = (1, 1)
-                self.padding2d = (0, 0)
-                self.dilation2d = (1, 1)
+                self.dim = dim
+                self.weight = torch.nn.Parameter(weight)
+                self.stride = (1,) * dim
+                self.padding = (0,) * dim
+                self.dilation = (1,) * dim
                 self.groups = 1
 
             def forward(self, x):
-                x = F.conv2d(
-                    x, self.weight2d, self.bias2d, self.stride2d, self.padding2d,
-                    self.dilation2d, self.groups)
+                x = convs[dim](
```

Comment:
can we assign it in the constructor and call `self.conv_fun` in the forward? it would just be easier to read the code this way